back-end/migrations/1575440239684_update_table_ProductModel.js

```diff
@@ -16,7 +20,25 @@ module.exports = {
   WHEN 2 THEN 'Unsecured'
   WHEN 3 THEN 'Pay day'
   WHEN 4 THEN 'Secured'
+  END,
+  duration = CASE id
+  WHEN 1 THEN '5 years fixed'
+  WHEN 2 THEN '6 months fixed'
+  WHEN 3 THEN '30 days fixed'
+  WHEN 4 THEN '10 years fixed'
+  END,
+  interest = CASE id
+  WHEN 1 THEN 3
+  WHEN 2 THEN 20
+  WHEN 3 THEN 15
+  WHEN 4 THEN 10
+  END,
+  title = CASE id
+  WHEN 1 THEN '3% yearly interest rate'
+  WHEN 2 THEN '20% monthly interest rate'
+  WHEN 3 THEN '15% daily interest rate'
+  WHEN 4 THEN '10% yearly interest rate'
```

Comment:
@JialinWang1 this is misleading all interest rate is yearly, just applying it happens differently, let's rephrase:
`3% interest, applied yearly` etc...